File-Formats/DocIO/Find-item-in-word-document.md

```diff
@@ -0,0 +1,738 @@
+---
+title: Find item in Word document | DocIO | Syncfusion
```

Comment:
Find item in Word document in .NET Word library | Syncfusion

Reply:
Addressed